### DISCUSSION TITLE
Is there a way for ArgoCD to have multiple git repo as source?

### DISCUSSION THREAD

**Discussion by Ian-Abellana** [argoproj/argo-cd#8836] [Q&A]
Hi all. I'm trying to find a way to separate the configmap from the rest of the manifest files so only the repo with the configmap is accessible by the developers and the manifest files with the ingress, deploy, etc are only accessible by the operations but so far, I am not able to see a resource where this can be supported. Is this a limitation of git as a source? Can this be done via helm? Looking forward to any suggestions/answers!

**Comment by alexmt** [comment]
The only option right now is to use config management plugin features. For example, with kustomize you might use [remote targets](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md). Helm child might reference charts from other repositories.


We are considering supporting multiple sources natively. The proposal is in progress: https://github.com/argoproj/argo-cd/pull/8322
